examples/ingest/local/ingest.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# Processes the Unstructured-IO/unstructured repository
+# through Unstructured's library in 2 processes.
+
+# Structured outputs are stored in local-ingest-output/
+
+# NOTE, this script is not ready-to-run!
+# You must enter
+#   1) --local-input-path  : path in the local file system which is to be processed
+#   2) --local-file-glob   : types of local files that are accepted,
+#                            provided as a comma-separated list
+# before running.
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+cd "$SCRIPT_DIR"/../../.. || exit 1
+
+PYTHONPATH=. ./unstructured/ingest/main.py \
+    --local-input-path "<path in local file system>" \
```

Comment:
```suggestion
    --local-input-path . \
```
it's ok to be opinionated here